learns-app-content/week-03/objectives.md

```diff
@@ -1,23 +1,33 @@
 <!-- h1, h2 already used by CTD Learns -->
 ### Topics Covered
 
-- topic 1
-- topic 2
-- topic n
+- Component lifecycle
+- Basic Hooks
+- State
+- Props
 
 ### Lesson Objectives
 
-#### Objective 1: Topic 1
+By the end of this lesson, we will:
+
+#### Objective 1: Component Lifecycle
 
 <!-- - {{blooms action verb}} sub-topic takeaway -->
 <!-- - {{blooms action verb}} sub-topic takeaway -->
 
-#### Objective 2: Topic 2
+#### Objective 2: Basic Hooks
 
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-<!-- - {{blooms action verb}} sub-topic takeaway -->
+- Explain the motivation behind React Hooks and the problems they solve
+- Understand the rules of hooks
 
-#### Objective 3: Topic 3
+#### Objective 3: State
 
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-<!-- - {{blooms action verb}} sub-topic takeaway -->
+- Explain the importance of state in managing dynamic data within React components
+- Demonstrate the usage of state with the useState hook
+
+#### Objective 4: Props
+
+- Understand props in React and how they facilitate parent-child component communication
```

Comment:
Does this encompass unidirectional data flow also?

Reply:
I'll change that to parent **to** child communication